scripts/build-products.js

```diff
@@ -110,6 +110,7 @@ const fetchProducts = async (coverProducts, provider) => {
         coverAssets: parseProductCoverAssets(coverAssets, coverAssetsMap),
         isPrivate,
         timestamp,
+        minPrice: minPrice || 100, // default to 1% if minPrice is not set
```

Comment:
(nit) for numbers the represent basis points, would it be better to put `_` to denote this? i.e. `1_00`

That way it kinda looks like 1.00% ~ `1_00`